charts/git-server/chart/values.yaml

```
@@ -1,24 +1,30 @@
-replicaCount: 3
+env: test
+replicaCount: 1
+nginx:
+  image:
+    repository: nginx
+    pullPolicy: IfNotPresent
+    tag: "1.21.4"
 
-image:
-  repository: ghcr.io/distributed-technologies/services-git-server
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "feature-git-server"
+fcgi:
+  image:
+    repository: git-server-fcgi
+    pullPolicy: IfNotPresent
+    tag: "0.17.0"
```

Should be (I think):
```suggestion
    repository: ghcr.io/distributed-technologies/helm-charts/git-server
    pullPolicy: IfNotPresent
    tag: "0.2.2"
```